Mathlib/FieldTheory/IsSepClosed.lean

```diff
@@ -191,6 +198,19 @@ class IsSepClosure [Algebra k K] : Prop where
 instance IsSepClosure.self_of_isSepClosed [IsSepClosed k] : IsSepClosure k k :=
   ⟨by assumption, isSeparable_self k⟩
 
+-- TODO: depends on `Algebra.IsAlgebraic.perfectField`
+/-- If `k` is perfect, `K` is a separable closure of `k`,
+then it is also an algebraic closure of `k`. -/
+proof_wanted IsSepClosure.isAlgClosure_of_perfectField
+    [Algebra k K] [IsSepClosure k K] [PerfectField k] : IsAlgClosure k K
```

Comment:
We should add `[PerfectField K] [IsSepClosed K] : IsAlgClosed K` too. That need not mention `k` but once combined with `Algebra.IsAlgebraic.perfectField` gives this immediately.

Reply:
I already added this, it's called `isAlgClosed_of_perfectField` at https://github.com/leanprover-community/mathlib4/pull/9488/files#diff-65948e34f88316c627865217c5250797373887972cda16a040e4b044067aac93R99